product-web-app/controller/capabilityController.ts

```diff
@@ -0,0 +1,19 @@
+import { Application, Request, Response } from 'express';
+import CapabilityService from '../service/capabilityService.js';
+import Capability from '../model/capability.js';
+
+export default class CapabilityController {
+  private capabilityServiceClass = new CapabilityService();
```

Comment:
```suggestion
  private capabilityService = new CapabilityService();
```
or if for some reason you want less confusion here it's capabilityServiceObject (but that's not a great name either)